pyproject.toml

```diff
@@ -0,0 +1,101 @@
+[project]
+name = "rtc-tools-heat-network"
+dynamic = ["version"]
+authors = [{ name = "Jim Rojer", email = "jim.rojer@tno.nl" }]
+description = "RTC Tools Heat Network is used for ESDLS containing a heat network. It can optimize and low-fidelity simulate OPEX and CAPEX costs as well as provide the optimal asset placements."
+classifiers = [
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Operating System :: OS Independent",
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: Information Technology",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Mathematics",
+    "Topic :: Scientific/Engineering :: Physics",
+]
+
+dependencies = [
+    "influxdb >= 5.3.1",
+    "pyecore",
+    "pymoca >= 0.9.0",
+    "rtc-tools == 2.6.0a3",
+    "pyesdl >= 21.11.0",
+    "pandas >= 1.3.1, < 2.0",
+]
+
+[project.optional-dependencies]
+dev = [
+    #"pip-tools~=6.13.0",  # Installed via create_env script
+    "black ~= 22.1.0",
+    "flake8 ~= 6.0.0",
+    "Flake8-pyproject ~= 1.2.3",
+    # "flake8-docstrings ~= 1.7.0", TODO Add in when docstrings are being added
```

Comment:
Turned off linting for docstrings as it is very noisy and not a priority at the moment. Added issue to capture future work on docs: https://github.com/Project-OMOTES/rtc-tools-heat-network/issues/68

Reply:
can we include flake8-bugbear, flake8-comprehensions, flake8-import-order like we have in the current master?